libs/basics/src/random.ts

```diff
@@ -0,0 +1,24 @@
+import crypto from 'crypto';
+
+import { assert } from './assert';
+
+/**
+ * Returns a random integer between min (inclusive) and max (inclusive). Intentionally uses the
+ * Node crypto library instead of Math.random to ensure that it's cryptographically secure.
+ */
+export function getRandomInteger({
+  min,
+  max,
+}: {
+  min: number;
+  max: number;
+}): number {
+  assert(Number.isInteger(min), 'min should be an integer');
+  assert(Number.isInteger(max), 'max should be an integer');
+  assert(min <= max, 'min should be less than or equal to max');
+
+  const range = max - min;
+  const byteArray = crypto.randomBytes(4); // 4 bytes for a 32-bit integer
```

Comment:
Is there a reason not to use [`crypto.randomInt()`](https://nodejs.org/docs/latest-v16.x/api/crypto.html#cryptorandomintmin-max-callback)?

Reply:
this is a great comment, not sure how I missed this!

Reply:
Ah yes good catch! https://github.com/votingworks/vxsuite/pull/3972/commits/05f279b03b9032ccf863981a7d6ab9dfa554f28b